sql/catalyst/src/main/scala/org/apache/spark/sql/internal/StaticSQLConf.scala

```diff
@@ -96,6 +96,14 @@ object StaticSQLConf {
     .toSequence
     .createOptional
 
+  val STREAMING_QUERY_LISTENERS = buildStaticConf("spark.sql.streamingQueryListeners")
```

Comment:
maybe -> `spark.sql.streaming.streamingQueryListeners ` for consistency.

Reply:
ok makes sense. renamed.